recipes/buffer-atob-btoa/src/workflow.ts

```diff
@@ -0,0 +1,57 @@
+import type { Edit, Range, SgRoot } from '@codemod.com/jssg-types/main';
+import type Js from '@codemod.com/jssg-types/langs/javascript';
+import { getNodeRequireCalls } from '@nodejs/codemod-utils/ast-grep/require-call';
+import { getNodeImportStatements } from '@nodejs/codemod-utils/ast-grep/import-statement';
+import { removeLines } from '@nodejs/codemod-utils/ast-grep/remove-lines';
+
+export default function transform(root: SgRoot<Js>): string | null {
+  const rootNode = root.root()
+  const edits: Edit[] = [];
+  const linesToRemove: Range[] = [];
+
+  const requireStatements = getNodeRequireCalls(root, 'buffer');
+  const importStatements = getNodeImportStatements(root, 'buffer');
+  const atobFunctionCalls = rootNode.findAll({
+    rule: {
+      pattern: 'buffer.atob($ARG)'
+    }
+  });
```

Comment:
To check the usage of the `atob` function, you first need to discover how it is used in this file, since it can be imported in many different ways, e.g.:

```js
import Buffer from "node:buffer"
import AnyRandomName from "node:buffer"
import { btoa } from "node:buffer"
import { btoa as reanamedImport } from "node:buffer"
const Buffer = require("node:buffer")
const atob = require("node:buffer").atob
const { atob } = require("node:buffer")
// and others
```

Happily, we have a utility function that can solve it for you `utils/ast-grep/resolve-binding-path.ts`

```suggestion
  const bind = resolveBindingPath(importNode, "$.atob")
  const atobFunctionCalls = rootNode.findAll({
    rule: {
      pattern: `${bind}($ARG)`
    }
  });
```